sxt/proof/sumcheck/BUILD

```diff
@@ -60,6 +60,42 @@ sxt_cc_component(
     ],
 )
 
+sxt_cc_component(
+    name = "partial_polynomial_mapper",
+    test_deps = [
+        "//sxt/algorithm/base:mapper",
+        "//sxt/base/test:unit_test",
+        "//sxt/scalar25/operation:overload",
+    ],
+    deps = [
+        ":polynomial_utility",
+        "//sxt/base/macro:cuda_callable",
+        "//sxt/scalar25/operation:add",
```

Comment:
Do the polynomial wrappers need an `add` dependency?

Reply:
Dropped. Thank you.